cmd/snapctl/task.go

```diff
@@ -198,7 +206,11 @@ func createTaskUsingWFManifest(ctx *cli.Context) error {
 
 	// Deadline for a task
 	dl := ctx.String("deadline")
-	maxFailures := ctx.Int("max-failures")
+	maxFailuresStr := ctx.String("max-failures")
+	maxFailures, err := stringValToInt(vmaxFailuresStr)
+	if err != nil {
+		return fmt.Errorf("Value for command-line option 'max-failures' (%v) cannot be parsed as an integer\n", maxFailuresStr)
+	}
```

Comment:
It seems like the whole point of having stringValToInt() instead of using strconv.Atoi directly is to provide a customized error message, but then here we discard that error message. Is there value in using stringValToInt over strconv.Atoi since this is the case?